src/openforms/submissions/form_logic.py

```diff
@@ -93,8 +97,10 @@ def evaluate_form_logic(
 
     updated_step_data = deepcopy(step.data)
     updated_submission_data = deepcopy(data)
+    evaluated_rules = list()
```

Comment:
```suggestion
    evaluated_rules = []
```
for consistency with the rest of the code